pom.xml

```diff
@@ -61,6 +61,32 @@
             <groupId>javax.servlet</groupId>
             <artifactId>jstl</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-security</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.security</groupId>
+            <artifactId>spring-security-taglibs</artifactId>
+        </dependency>
+<!--        <dependency>-->
```

Comment:
Warto usuwać zakomentowany kod, albo zrobić commit, potem usunąć kod, zamiast komentować i kolejny commit, kiedy będziesz chciał przywrócić kod to zrobisz operacje revert commit i w łatwy sposób zmiany wrócą ;)